src/NuGet.Clients/PackageManagement.UI/Models/SearchResultPackageMetadata.cs

```diff
@@ -104,4 +217,13 @@ public override string ToString()
             return Id;
         }
     }
-}
+
+    internal class BackgroundLoaderResult
+    {
+        public PackageStatus Status { get; set; }
+
+        public NuGetVersion LatestVersion { get; set; }
+
+        public NuGetVersion InstalledVersion { get; set; }
+    }
+}
```

Comment:
The background loader now needs to load more data, no just Status. So created this internal class to represent the result returned by the background loader